### ISSUE TITLE
Can't get this to work on new HASS install (RPi)

### ISSUE THREAD

**Issue by prezmc** [itchannel/fordpass-ha#157]
I just setup an RPi 3 with HASS, I got HACS installed.  I found/added the custom repo for Fordpass HA, and HACS and HASS know it's there.

That's where progress stops.  The instruction say:

 Once the integration is installed **go to your integrations and follow the configuration options to specify the below:

Username (Fordpass App)
Password (Fordpass App)
VIN Number
Region (Where you are based, required for tokens to work correctly)**

This doesn't work.  When I go to the normal HASS integrations, or even the integrations page in HACS, and search for "Ford..." nothing is found to add, to do any config with.

Help!

Thanks,
Shannon
<img width="1787" alt="Screen Shot 2022-01-10 at 8 47 43 PM" src="https://user-images.githubusercontent.com/97483214/148872722-f2fe33cc-72b4-40e7-9b83-36e102439697.png">
<img width="1788" alt="Screen Shot 2022-01-10 at 8 47 32 PM" src="https://user-images.githubusercontent.com/97483214/148872746-338c9157-40aa-4300-a885-7d9f0d892f30.png">

**Comment by prezmc** [comment]
Update:  Patience is a virtue I still don't have?

I scrolled through closed issues, and found another with a similar problem.  I waited a little while longer, and it showed up.  I have authenticated and tested the remote start.  So far so good, now to build a custom card for it.  

It might be good to add a note in the instructions letting people know that it may take a while to show up correctly after downloaded/installed? (Since I am not the only one to see that behavior).

Thanks for this cool integration
